src/Search/Microsoft.Azure.Search/Customizations/Indexes/Models/AnalyzerName.cs

```diff
@@ -466,31 +466,515 @@ public sealed class AnalyzerName : ExtensibleEnum<AnalyzerName>
+
+            /// <summary>
+            /// Microsoft analyzer for Slovak.
+            /// </summary>
+            public const string SkMicrosoft = "sk.microsoft";
+
+            /// <summary>
+            /// Microsoft analyzer for Slovenian.
+            /// </summary>
+            public const string SlMicrosoft = "sl.microsoft";
+
+            /// <summary>
+            /// Microsoft analyzer for Spanish.
+            /// </summary>
+            public const string EsMicrosoft = "es.microsoft";
+
+            /// <summary>
+            /// Lucene analyzer for Spanish.
+            /// </summary>
+            public const string EsLucene = "es.lucene";
+
+            /// <summary>
+            /// Microsoft analyzer for Swedish.
+            /// </summary>
+            public const string SvMicrosoft = "sv.microsoft";
+
+            /// <summary>
+            /// Lucene analyzer for Swedish.
+            /// </summary>
+            public const string SvLucene = "sv.lucene";
+
+            /// <summary>
+            /// Microsoft analyzer for Tamil.
+            /// </summary>
+            public const string TaMicrosoft = "ta.microsoft";
+
+            /// <summary>
+            /// Microsoft analyzer for Telugu.
+            /// </summary>
+            public const string TeMicrosoft = "te.microsoft";
+
+            /// <summary>
+            /// Microsoft analyzer for Thai.
+            /// </summary>
+            public const string ThMicrosoft = "th.microsoft";
+
+            /// <summary>
+            /// Lucene analyzer for Thai.
+            /// </summary>
+            public const string ThLucene = "th.lucene";
+
+            /// <summary>
+            /// Microsoft analyzer for Turkish.
+            /// </summary>
+            public const string TrMicrosoft = "tr.microsoft";
+
+            /// <summary>
+            /// Lucene analyzer for Turkish.
+            /// </summary>
+            public const string TrLucene = "tr.lucene";
+
+            /// <summary>
+            /// Microsoft analyzer for Ukranian.
+            /// </summary>
+            public const string UkMicrosoft = "uk.microsoft";
+
+            /// <summary>
+            /// Microsoft analyzer for Urdu.
+            /// </summary>
+            public const string UrMicrosoft = "ur.microsoft";
+
+            /// <summary>
+            /// Microsoft analyzer for Vietnamese.
+            /// </summary>
+            public const string ViMicrosoft = "vi.microsoft";
+
+            /// <summary>
+            /// Standard Lucene analyzer.
+            /// </summary>
+            public const string StandardLucene = "standard.lucene";
+
+            /// <summary>
+            /// Standard ASCII Folding Lucene analyzer.
+            /// <see href="https://msdn.microsoft.com/library/azure/mt605304.aspx#Analyzers" /> 
+            /// </summary>
+            public static readonly AnalyzerName StandardAsciiFoldingLucene =
```

Comment:
Fixed in https://github.com/Azure/azure-sdk-for-net/pull/2536/commits/f46c44cde80ec93d73b15c292ff31f1ca72d531c (resquashed)